packages/netlify-cms-widget-markdown/src/serializers/remarkShortcodes.js

```diff
@@ -10,17 +10,16 @@ export function remarkParseShortcodes({ plugins }) {
 
 function createShortcodeTokenizer({ plugins }) {
   return function tokenizeShortcode(eat, value, silent) {
-    let match;
-    const potentialMatchValue = value.split('\n\n')[0].trimEnd();
```

Comment:
We once removed this code and had to revert it.
See https://github.com/netlify/netlify-cms/pull/3066/files#diff-48d1e9e4138a1ae21bdcb6be3721605365ec4a5a5c23de91c6ee8c448c4f830cL13
and https://github.com/netlify/netlify-cms/pull/3089/files#diff-48d1e9e4138a1ae21bdcb6be3721605365ec4a5a5c23de91c6ee8c448c4f830cR14.

Removing it breaks the built-in image component, see:
![image](https://user-images.githubusercontent.com/26760571/115147013-b7bfbc00-a061-11eb-9c72-ba4e951988a6.png)

Reply:
Huh! Thanks for catching. I'll look into this and make sure I come up with something that does not break images!

Reply:
Good news: It seems that the rest of the refactor has resolved whatever issue required this potentialMatchValue hack. Here's the kitchen sink with my change:

<img width="822" alt="image" src="https://user-images.githubusercontent.com/5354254/124860719-28e77f00-df80-11eb-9760-b80eaed632b0.png">

Images seem fine!